docs/design.md

```diff
@@ -149,7 +149,8 @@ The `Logic` component handles the business logic of TEAMMATES. In particular, it
 
 Package overview:
 - **`logic.api`**: Provides the API of the component to be accessed by the UI.
-- **`logic.core`**: Contains the core logic of the system.
+- **`logic.core`**: Contains the core logic of the system and its internal data entities.
```

Comment:
> and its internal data entities

This is saying `logic.core` contains the internal data entities, which obviously is not the case. This layer does concern the internal data entities, but is not the layer that contains it.